nuxeo-filters-behavior.html

```diff
@@ -15,6 +15,7 @@
 limitations under the License.
 -->
 <link rel="import" href="es6-polyfills.html">
+<link rel="import" href="nuxeo-i18n-behavior.html">
```

Comment:
This doesn't seem to be used.

Reply:
Indeed. Were you able to understand why it is there?

Reply:
Nope...